### PR TITLE
`pj-rehearse` plugin: add rehearsals-ack label

### PR DESCRIPTION
**Pull request by smg247**
We need the label for "phase two" where it is required to merge

/cc @openshift/test-platform @droslean

### CODE REVIEW

File: core-services/prow/02_config/_labels.yaml [hunk 1]
```diff
@@ -587,6 +587,12 @@ repos:
         target: both
         prowPlugin: label
         addedBy: anyone
+      - color: 5df542
```
Comment (droslean) [inline core-services/prow/02_config/_labels.yaml:590]: I think it would be better to change this in a different PR. And after the `label-sync` job succeeds, you can add the label to tide's config.

**Review by hongkailiu — APPROVED** [review]
